packages/blade/src/tokens/global/motion.ts

```diff
@@ -0,0 +1,132 @@
+import makeBezier from '../../utils/makeBezier';
+
+export type Motion = Readonly<{
+  delay: {
+    /** `70` milliseconds */
```

Comment:
Man!! I really wish typescript had a way to derive these values from the code. If we change the value this wont be updated. :(

Reply:
Yes :(